src/plugins/intel_gpu/src/graph/include/program_node.h

```diff
@@ -146,6 +146,10 @@ struct program_node {
                                                                                  get_fused_primitives(),
                                                                                  get_fused_activations_funcs(), get_fused_activations_params()));
         params->memory_deps = get_const_memory_deps();
+        params->constant_deps.clear();
+        for (auto& in : get_dependencies()) {
+            params->constant_deps.push_back(in->is_constant());
+        }
```

Comment:
How about add additioanl info to memory_deps whthere it is constnat or not, instead of having additional variable?

Reply:
Create primary_input_idx instead of constant_deps in kernel param.